### PR TITLE
Add the refresh after install functionality to the plugin

### PR DESCRIPTION
**Pull request by sebbeaume**
The functiononality is in AdapterJS.renderNotificationBar, from line 352.

### CODE REVIEW

**Comment by johache** [comment]
1 - Please replace tabs with double spaces 
2 - the pull request should go from your branch to development, not master

File: publish/adapter.debug.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ AdapterJS.WebRTCPlugin.defineWebRTCInterface = null;
 AdapterJS.WebRTCPlugin.isPluginInstalled = null;
 
  // Lets adapter.js wait until the the document is ready before injecting the plugin
-AdapterJS.WebRTCPlugin.pluginInjectionInterval = null;
+ AdapterJS.WebRTCPlugin.pluginInjectionInterval = null;
```
Comment (johache) [inline publish/adapter.debug.js]: Removed space, avoid meaningless changes.

File: publish/adapter.debug.js [hunk 2]
```diff
@@ -123,7 +123,7 @@ Log outputs are prefixed in some cases.
   INFO: Information reported by the plugin.
   ERROR: Errors originating from within the plugin.
   WEBRTC: Error originating from within the libWebRTC library
-*/
+  */
```
Comment (johache) [inline publish/adapter.debug.js]: Removed space, avoid meaningless changes.

File: publish/adapter.debug.js [hunk 3]
```diff
@@ -294,7 +294,14 @@ AdapterJS.addEvent = function(elem, evnt, func) {
...
 
-AdapterJS.renderNotificationBar = function (text, buttonText, buttonLink, openNewTab, displayRefreshBar) {
+
+AdapterJS.reloadPage = function()
+{
+  alert("Plug-in installed.");
+  location.reload();
+}
+
+AdapterJS.renderNotificationBar = function (text, buttonText, buttonLink, openNewTab, displayRefreshBar, func) {
```
Comment (johache) [inline publish/adapter.debug.js]: func is not clear enough

**Comment by johache** [comment]
@sebbeaume for reference, have a look at the 2 commits I just added.
Question: given that we reload the plugin list, could we just do the plugin injection from onplugininstalled ? Would the plugin work in the case? That would save us a page reload.
Also please change this PR to development and not master.

**Comment by sebbeaume** [comment]
@johache  I tried to use the injectPlugin function. It gives an error "getUserMedia unknown". That doesn't happen if page is reloaded.
Changing branches is impossible once the PR has been made. I have to close the request and to make a new one with the correct target.

**Pull request closed without merging**